components/brave_rewards/browser/contribution_publisher.cc

```diff
@@ -0,0 +1,23 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/components/brave_rewards/browser/contribution_publisher.h"
+
+namespace brave_rewards {
+
+ContributionPublisher::ContributionPublisher()
+    : total_amount_(0.0), contributed_amount_(0.0) {}
+
+ContributionPublisher::~ContributionPublisher() {}
+
+ContributionPublisher::ContributionPublisher(
```

Comment:
Is this just the default copy constructor implementation? If so, since it's a struct can we just let the compiler generate it?

Reply:
what we can do here is `ContributionPublisher::ContributionPublisher( const ContributionPublisher& info) = default;`

Reply:
Fixed.